server/mscalendar/views/status_change_notification.go

```diff
@@ -49,7 +49,7 @@ func RenderEventWillStartLine(subject, weblink string, startTime time.Time) stri
 
 func renderScheduleItem(event *remote.Event, status string) string {
 	if event == nil {
-		return fmt.Sprintf("You have no upcoming events.\n Shall I change your status to %s?", prettyStatuses[status])
+		return fmt.Sprintf("You have no upcoming events.\n Shall I change back your status to %s?", prettyStatuses[status])
```

Comment:
0/5
```suggestion
		return fmt.Sprintf("You have no upcoming events.\n Shall I change your status back to %s?", prettyStatuses[status])
```

Reply:
You are the native speaker, so I will go with your suggestion.